buildspecs/build.yml

```diff
@@ -15,4 +15,11 @@ phases:
           cd test/module-path-tests
           mvn package
           mvn exec:exec -P mock-tests
-        fi
\ No newline at end of file
+        fi
+    finally:
+      - mkdir -p codebuild-test-reports
+      - find ./ -name 'TEST-*.xml' -type f -exec cp {} codebuild-test-reports/ \;
```

Comment:
Isn't there HTML output as well?

Reply:
The reports only process the XML files

Reply:
Ah, cool, I didn't know that they generated dashboards.